apache/superset
# refactor: Updates some database columns to MediumText

### SUMMARY
Similarly to https://github.com/apache/superset/pull/24911, https://github.com/apache/superset/pull/24510, https://github.com/apache/superset/pull/20936, https://github.com/apache/superset/pull/20779, https://github.com/apache/superset/pull/5618, this PR updates the type of some database columns to `MediumText` to accommodate bigger text contents such as JSON, CSS or SQL. We have found some cases where SQL queries shared by users were truncated due to insufficient storage space. You can check the referenced PRs for more context on why the change is important.

### TESTING INSTRUCTIONS
1 - Execute `superset db upgrade` and check the affected tables
2 - Make sure the nullable state is preserved
3 - Execute `superset db downgrade` and check the affected tables

### ADDITIONAL INFORMATION
